src/screens/Elections/ScreenContainer.js

```diff
@@ -1,30 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Colors from '../../styles/colors';
 import { connect } from 'react-redux';
-import { compose, lifecycle } from 'recompose';
-import { getCandidates, loadCandidates } from '../../candidate/redux/candidates';
-import { loadUser } from '../../user/sagas'
-import { getIsLoggedIn } from '../../auth/selectors';
-import WithAuthentication from '../../util/components/WithAuthentication';
-import Screen from './Screen';
+import {StyleSheet, ScrollView, View, Text, FlatList, TouchableOpacity } from 'react-native';
+import { getCandidateData } from './viewSelectors'
+import DataContainer from './CandidateContainer'
 
-const ScreenWithAuthentication = WithAuthentication('logout')(Screen);
+const ElectionsScreen = props => {
+  return (
+    <View style={styles.container}>
+      <FlatList
+        data={props.elections.elections}
+        keyExtractor={(item)=>item.electionIds}
+        renderItem={({item})=> (
+          <View key={item} style={styles.candidateContainer}>
+            <View style={styles.informationContainer}>
+              <Text style={styles.positionText}>{item.electionIds} </Text>
+              <Text style={styles.electionText}>November 6, 2018</Text>
+            </View>
+            <ScrollView horizontal={true}>
+              {item.candidates.map(candidate => { 
+                return ( 
+                  <TouchableOpacity key={candidate.id} onPress={ props.goToCandidateDetail(candidate.id)}>
+                    <Candidate candidateId={candidate.id}/>
+                  </TouchableOpacity>
+                ) 
+              })}
+            </ScrollView>
+          </View>
+        )}
+      />
+    </View>
+  );
+}
 
-const Container = compose(
-  connect(
-    state => ({
-      candidates: getCandidates(state, toListCandidateMapperPlaceholder),
-      isLoggedIn: getIsLoggedIn(state),
-    }),
-    { loadCandidates, loadUser },
-  ),
-  lifecycle({
-    componentDidMount() {
-      this.props.loadCandidates();
-      // TODO: only load user if match data has not been loaded yet
-      this.props.loadUser();
-    },
+ElectionsScreen.propTypes = {
+  goToCandidateDetail: PropTypes.func,
+  candidates: PropTypes.array,
```

Comment:
Are you still using the candidates here? I think they are in the elections, but I might have missed something

Reply:
thanks for the catch!
*removed*